src/scanner.cc

```diff
@@ -94,17 +169,13 @@ struct Scanner {
     }
 
     expected_percent_count.push_back(count);
-
-    // A START is fully scanned when we started with an 'm' (precondition of
```

Comment:
Could we keep the information and in particular precondition and postcondition, although I imagine that they need to be updated?

Reply:
I've added a little bit of explanation into the function. With this change there is actually no precondition for calling `scan_multstr_start` anymore.